modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -25,22 +25,35 @@ function perflab_aea_audit_enqueued_scripts() {
 		global $wp_scripts;
 		$enqueued_scripts = array();
 
-		foreach ( $wp_scripts->queue as $handle ) {
-			$src = $wp_scripts->registered[ $handle ]->src;
-			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
-				$enqueued_scripts[] = array(
-					'src'  => $src,
-					'size' => perflab_aea_get_resource_file_size( perflab_aea_get_path_from_resource_url( $src ) ),
-				);
+		foreach ( $wp_scripts->done as $handle ) {
+			$script = $wp_scripts->registered[ $handle ];
+
+			if ( ! $script->src || false !== strpos( $script->src, 'wp-includes' ) ) {
+				continue;
+			}
+
+			// Add any extra data (inlined) that was passed with the script.
+			$inline_size = 0;
+			if ( ! empty( $script->extra ) && ! empty( $script->extra['after'] ) ) {
+				foreach ( $script->extra['after'] as $extra ) {
+					$inline_size += ( is_string( $extra ) ) ? mb_strlen( $extra, '8bit' ) : 0;
+				}
 			}
+
+			$path               = perflab_aea_get_path_from_resource_url( $script->src );
```

Comment:
Since you've changed `perflab_aea_get_path_from_resource_url()` to support empty values (which may not be necessary), we need to make sure this here can also deal with that change properly, supporting an empty return value.
```suggestion
			$path = perflab_aea_get_path_from_resource_url( $script->src );
			if ( ! $path ) {
				continue;
			}

```